src/config.rs

```diff
@@ -1289,6 +1294,38 @@ impl Dimensions {
     }
 }
 
+/// Pixels per inch
+///
+/// This is only used on `FreeType` systems
+#[derive(Debug, Deserialize)]
+pub struct Dpi {
```

Comment:
This doesn't appear to be used anywhere else in the codebase. I think it was removed in https://github.com/jwilm/alacritty/commit/2ea20f4823ae96f92f47a1984a6dd118a9632fdb..